README.md

```diff
@@ -21,34 +21,13 @@ Start a new [KMM](https://kotlinlang.org/docs/mobile/create-first-app.html) proj
 
 *See [Config.kt](buildSrc/src/main/kotlin/Config.kt#L2txt) or the [realm-kotlin releases](https://github.com/realm/realm-kotlin/releases) for the latest version number.*
 
-- Add the following Gradle configuration in the root project (make sure you're using Kotlin `1.4.20` or recent)
-`<root project>/build.gradle.kts`
-```Gradle
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20")// minimum 1.4.20
-        classpath("com.android.tools.build:gradle:4.0.1")
-        classpath("io.realm.kotlin:gradle-plugin:<VERSION>")
-    }
-}
-
-allprojects {
-    repositories {
-        mavenCentral()
-    }
-}
-```
-
-- Apply the `io.realm.kotlin` plugin and specify the dependency in the common source set.
+- In the shared module (`shared/build.gradle.kts`), apply the `io.realm.kotlin` plugin and specify the dependency in the common source set.
 
 ```Gradle
 plugins {
     kotlin("multiplatform")
     id("com.android.library")
-    id("io.realm.kotlin")
+    id("io.realm.kotlin") version "<VERSION>"
```

Comment:
Is there a good reason to include the version here as well? In my experience it's just another place to update when you want to increase your version. I'm currently leaving it out in the official docs, but if this is something useful I can always add it.

Reply:
It depends on how you applied the plugin (see https://plugins.gradle.org/plugin/io.realm.kotlin) the legacy way doesn't need to specify the version since it's been provided in the `buildscript` dependency. In this PR we focus on the modern way of applying the plugin